src/schema/SearchableItem/SearchableItemPresenter.ts

```diff
@@ -0,0 +1,119 @@
+import moment from "moment"
+import { stripTags } from "lib/helpers"
+
+const DATE_FORMAT = "MMM Do, YYYY"
+
+export class SearchableItemPresenter {
```

Comment:
Nice, I like the stand-alone module (and a presenter class to boot!) to encapsulate this logic.